File: Basic_Level_C#/C#_Projects_Visual_Studio/Homework04ConsoleInputOutput/09.SumOfNumbers/SumOfNumbers.cs
using System;

class SumOfNumbers
{
    static void Main()
    {
        int countOfNumbers = int.Parse(Console.ReadLine());
        double sum = 0;

        for (double i = 1; i <= countOfNumbers; i++)
        {
            double number = double.Parse(Console.ReadLine());
            sum += number;
        }

        Console.WriteLine(sum);
    }
}

File: Fundamental_Level_Java/01.Java_Fundamentals_Mar_2016/01.Homework/src/SumOfNumbers.java
import java.util.Scanner;

public class SumOfNumbers {

    public static void main(String[] args) {

        Scanner input = new Scanner(System.in);

        int number = input.nextInt();
        long sum = 0;

        for (int i = 1; i <= number; i++) {
            sum += i;
        }
        System.out.println(sum);
        input.close();
    }
}
